retrieval/discovery/triton.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
Duplicate file for the location before we moved `discovery` into a top-level package?